index.php

```diff
@@ -9,7 +9,13 @@
 			while ( $row = $comics->series_list_result->fetch_assoc () ) {
 				$series_id = $row ['series_id'];
 				$series_name = $row ['series_name'];
-				$series_list .= "<li><a href=\"issues.php?series_id=$series_id\">" . $series_name . "</a></li>\n";
+				$series_vol = $row ['series_vol'];
+				$comics->seriesLatestCover($series_id);
+				$series_cover = $comics->series_latest_cover;
+				if ($series_cover == NULL) {
+					$series_cover = 'assets/nocover.jpg';
+				}
+				$series_list .= '<li class="col-xs-6 col-sm-3 col-md-2"><a href="issues.php?series_id=' . $series_id . '" class="series-info"><img src="/' . $series_cover  . '" /><div class="series-title"><h3>' . $series_name . '</h3></div></a><small>Volume ' . $series_vol . '</small><a href="#" class="button add-button">[Add New]</a><a href="#" class="button edit-button">[Edit]</a></li>';
 			}
```

Comment:
It's responsive! Grow/shrink the browser window and watch the series columns resize!